wled00/button.cpp

```diff
@@ -284,10 +298,12 @@ void handleButton()
       buttonPressedBefore[b] = true;
 
       if (now - buttonPressedTime[b] > WLED_LONG_PRESS) { //long press
-        if (!buttonLongPressed[b]) longPressAction(b);
-        else if (b) { //repeatable action (~3 times per s) on button > 0
+        if (!buttonLongPressed[b]) {
+          buttonBriDirection = !buttonBriDirection; //toggle brightness direction on long press
+          longPressAction(b);
+        } else if (b) { //repeatable action (~5 times per s) on button > 0
           longPressAction(b);
-          buttonPressedTime[b] = now - WLED_LONG_REPEATED_ACTION; //333ms
+          buttonPressedTime[b] = now - WLED_LONG_REPEATED_ACTION; //200ms
```

Comment:
According to #define that is 400ms.

Reply:
The rationale behind this choice is that the higher the value the greater the number of repetitions per seconds, WLED_LONG_REPEATED_ACTION does not actually mean "repeat the action every XXX milliseconds", I don't know how to explain with words so I'll make this example:
- When WLED_LONG_REPEATED_ACTION = 300, suppose now = 1000:
```buttonPressedTime[b] = now - WLED_LONG_REPEATED_ACTION ----> buttonPressedTime[b] = 700```
next call of handleButton we assume now = 1201
```if (now - buttonPressedTime[b] > WLED_LONG_PRESS) ----> if(1201 - 700 > 600) ----> if(501 > 600)``` ---> repeated action DOES NOT trigger


- When WLED_LONG_REPEATED_ACTION = 400, suppose now = 1000:
```buttonPressedTime[b] = now - WLED_LONG_REPEATED_ACTION ----> buttonPressedTime[b] = 600```
next call of handleButton we assume now = 1201
```if (now - buttonPressedTime[b] > WLED_LONG_PRESS) ----> if(1201 - 600 > 600) ----> if(601 > 600)``` ---> repeated action DOES trigger

The greater the value of ```WLED_LONG_REPEATED_ACTION``` is, the greater we set ```buttonPressedTime[b]``` back in time, therefore increasing the number of repetitions per second, with the actual delay between repetitions being ```WLED_LONG_PRESS - WLED_LONG_REPEATED_ACTION``` 
(now - buttonPressedTime[b] > WLED_LONG_PRESS equals to 
now_new - now_old + WLED_LONG_REPEATED_ACTION > WLED_LONG_PRESS that is 
now_new - now_old > WLED_LONG_PRESS - WLED_LONG_PRESS)

Hope I did not make myself a fool by totally misunderstanding the code 😅

Reply:
Ok, IDK who implemented this (will need to check git blame, I hope it was not me) but it is un-intuitive. 😄 It looks like it is roughly 1000/WLED_LONG_REPEAT_ACTION*100 ms.